draft/2025-03-12-this-week-in-rust.md

```diff
@@ -45,6 +45,11 @@ and just ask the editors to select the category.
 
 ### Miscellaneous
 
+* Deadline extended for
+  [GOSIM Rust Spotlight](https://spotlight.gosim.org/rust2025)
+  [Nominations](https://cfp.gosim.org/spotlight-rust) ---
+  please support your favorite projects!
```

Comment:
```suggestion
* Deadline extended for
  [GOSIM Rust Spotlight](https://spotlight.gosim.org/rust2025#deadline-extended)
  [Nominations](https://cfp.gosim.org/spotlight-rust#deadline-extended) ---
  please nominate and support your favorite projects by 2025-03-15 at 7:59am UTC!
```

This specifies the actual deadline, and fixes the link URLs to not duplicate the ones from the previous issue.